recipe/meta.yaml

```diff
@@ -40,11 +40,17 @@ test:
     - quantecon.markov
     - quantecon.random
     - quantecon.util
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
 
 about:
-  home: https://github.com/QuantEcon/QuantEcon.py
+  home: https://quantecon.org/quantecon-py/
+  license_url: https://github.com/QuantEcon/QuantEcon.py/blob/v{{ version }}/LICENSE
+  license_family: BSD
   license: BSD-3-Clause
-  license_file: LICENSE.txt 
```

Comment:
Should be   `license_file: LICENSE `. But then remove `license_url`

Reply:
Great, will do. Thanks!